doc/devel/contributing.rst

```diff
@@ -48,6 +48,17 @@ use to organize this information.
 
 Thank you for your help in keeping bug reports complete, targeted and descriptive.
 
+Requesting a new feature
+========================
+
+Please post feature requests to the
+`Issue Tracker <https://github.com/matplotlib/matplotlib/issues>`_.
+
+The Matplotlib developers will give feedback on the feature proposal. After
+acceptance, the feature can be implemented. Since Matplotlib is an open source
```

Comment:
what does acceptance mean? this may be a longer discussion for the call, but should be pinned down.

Reply:
I want to express that not every feature request may be deemed an appropriate extension of Matplotlib by core developers. We welcome if somebody proposes a feature and also implements it, but it may be good to first get feedback before you start implementing.

I don't want to go into possible details of the feedback/approval/acceptance process here.

Thinking again, I think we can leave the sentence out.

Reply:
"If there is significant support, you can try implementing the feature"?

Reply:
Thanks for the proposal. I have the impression that it might fee a little intimidating. "significant" is a strong word and may be a barrier for people considering a propsal. Also, the phrasing might give the false impression that the proposer must also implement it.

I'm going with removing the sentence. I think it's clear enough; and it's also not the end of the world if a PR is opened before a feature is fully discussed. Sometimes you may even need it to illustrate the intention.